Update setReportEventDataForAutomaticBeacons (#448)
WICG/turtledove: Fenced_Frames_Ads_Reporting.md

});
```

> The new event, if registered, implies that an automatic beacon will be sent by the browser to the registered URL when a top-level navigation is invoked from within the fenced frame and the navigation was preceded by window.fence.setReportEventDataForAutomaticBeacons. This will impact top-level navigation initiated from the fenced frame in the same tab (via [unfencedTop target](https://github.com/WICG/fenced-frame/blob/master/explainer/integration_with_web_platform.md#top-level-navigation)) or in a different tab. Note that this beacon is gated on a transient user activation. More details about the beacon are below.

Reviewer: "preceded" -> "preceded by a call to"

Given that this is the first mention of setReportEventDataForAutomaticBeacons, can we link it to a definition?

Author: Done.

If invoked multiple times, the latest invocation before the top-level navigation would be the one that’s honored.

> eventData can be empty. In that case, the automatic beacon will still be sent but without an event data body in the HTTP request.

Reviewer: ```suggestion
`eventData` can be empty, in which case the automatic beacon will still be sent but without an event data body in the HTTP request.
```

> If this API is not invoked, destination for the automatic beacon will be empty and so no beacon request can be sent by the browser. 

Reviewer: ```suggestion
If `setReportEventDataForAutomaticBeacons` is not invoked, the browser will not send an automatic beacon because the `destination` is unknown.
```
